pkg/e2e/operators/machine-api-operator.go

```diff
@@ -23,9 +27,13 @@ var _ = g.Describe("[Feature:Operators] Machine API operator deployment should",
 		client, err := e2e.LoadClient()
 		o.Expect(err).NotTo(o.HaveOccurred())
 
-		deploymentName := "clusterapi-manager-controllers"
+		deploymentName := "machine-api-controllers"
 		initialDeployment, err := getDeployment(client, deploymentName)
-		o.Expect(err).NotTo(o.HaveOccurred())
+		if err != nil {
```

Comment:
`if apierrors.IsNotFound(err) {...} else {o.Expect(err).NotTo(o.HaveOccurred())}`

Reply:
`apierrors "k8s.io/apimachinery/pkg/api/errors"`

Reply:
realised the error is actually a polling time out